injector/src/main/java/edu/ucr/cs/riple/injector/Injector.java

```diff
@@ -70,8 +69,8 @@ public <T extends Change> Set<FileOffsetStore> start(Set<T> changes) {
         (path, changeList) -> {
           CompilationUnit tree;
           try {
-            tree = LexicalPreservingPrinter.setup(StaticJavaParser.parse(new File(path)));
-          } catch (FileNotFoundException exception) {
+            tree = LexicalPreservingPrinter.setup(StaticJavaParser.parse(path));
+          } catch (IOException exception) {
```

Comment:
Why are we switching to a more general exception here? Is there something more precise we check for? Also, is `LexicalPreservingPrinter` still in use?

Reply:
Yes, this method specifically throws a more general exception, also I just tested removing call to `LexicalPreservingPrinter` and everything is working as expected, I will remove it in a follow up PR.